VogtPizza/Display.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPizza
+{
+    class Display
```

Comment:
Consider calling this PizzaMenu or PizzaMenuDisplay.

Display is very very generic. Id expect it contain all console.WriteLine() given the name. But really, you're just printing out the menus here.